demisto_client/__init__.py

```diff
@@ -67,6 +71,9 @@ def configure(base_url=None, api_key=None, verify_ssl=None, proxy=None, username
     configuration.proxy = proxy
     configuration.debug = debug
     configuration.ssl_ca_cert = ssl_ca_cert
+    if connection_pool_maxsize and connection_pool_maxsize.isdigit():
```

Comment:
Would rather we raise an exception if we have a configuration which is not a number. Simply ignoring stuff is very hard to detect. Also this will raise if the arg connection_pool_maxsize was passed as an int.